var/spack/repos/builtin/packages/samurai/package.py

```diff
@@ -0,0 +1,63 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Samurai(CMakePackage):
+    """Intervals coupled with algebra of set to handle adaptive
+    mesh refinement and operators on it"""
+
+    homepage = "https://github.com/hpc-maths/samurai"
+    git = "https://github.com/hpc-maths/samurai.git"
+
+    version("master", branch="master")
+
+    variant("mpi", default=False, description="Enable MPI support")
+    variant("openmp", default=False, description="Enable OpenMP support")
+    # variant("demos", default=False, description="Build Demos")
+    # variant("benchmarks", default=False,description="Build benchmarks")
+    variant("tests", default=False, description="Build tests")
+    variant("check_nan", default=False, description="Check for Nan in computations")
+
+    depends_on("xtl@0.7.4")
+    # depends_on("xsimd@11.0.0")
+    depends_on("xtensor@0.24.1 ~tbb")
+    depends_on("highfive~mpi", when="~mpi")
+    depends_on("highfive+mpi", when="+mpi")
+    depends_on("pugixml")
+    depends_on("fmt")
+    depends_on("nlohmann-json")
+    depends_on("cli11")
+
+    # option in future release
```

Comment:
What does this option mean? Are all of these dependencies optional? Or just `cxxopts`?